flashpay/settings/local.py

```diff
@@ -9,9 +9,9 @@
 
 
 REST_FRAMEWORK = {
-    "DEFAULT_PERMISSION_CLASSES": ("rest_framework.permissions.AllowAny",),
+    "DEFAULT_PERMISSION_CLASSES": ("rest_framework.permissions.IsAuthenticatedOrReadOnly",),
     "DEFAULT_AUTHENTICATION_CLASSES": (
-        "rest_framework_simplejwt.authentication.JWTStatelessUserAuthentication",
+        "flashpay.apps.account.authentication.CustomJWTAuthentication",
```

Comment:
Since `REST_FRAMEWORK` for `settings/base.py` & `settings/local.py` are the same, there's no need to redefine it I guess

Reply:
okay 👍

Reply:
They are not the same. I removed the `Exception Handler` so the app can crash  on local testing